internal/bgp/frr/config.go

```diff
@@ -71,6 +71,7 @@ type neighborConfig struct {
 	Port                uint16
 	HoldTime            uint64
 	KeepaliveTime       uint64
+	ConnectTime         *uint64
```

Comment:
no need for it to be a pointer, I'd defer the ne 0 logic to the template. It's kind of equivalent but probably more straightforward

Reply:
i understand, changed it.